sdk/eventhub/azure-eventhub-checkpointstoretable/azure/eventhub/extensions/checkpointstoretable/_tablestoragecs.py

```diff
@@ -17,22 +56,299 @@ class TableCheckpointStore:
+         The format is like "<namespace>.servicebus.windows.net".
+        :param str eventhub_name: The name of the specific Event Hub the partition ownerships are associated with,
+         relative to the Event Hubs namespace that contains it.
+        :param str consumer_group: The name of the consumer group the ownerships are associated with.
+        :rtype: Iterable[Dict[str, Any]], Iterable of dictionaries containing partition ownership information:
+                - `fully_qualified_namespace` (str): The fully qualified namespace that the Event Hub belongs to.
+                  The format is like "<namespace>.servicebus.windows.net".
+                - `eventhub_name` (str): The name of the specific Event Hub the checkpoint is associated with,
+                  relative to the Event Hubs namespace that contains it.
+                - `consumer_group` (str): The name of the consumer group the ownership are associated with.
+                - `partition_id` (str): The partition ID which the checkpoint is created for.
+                - `owner_id` (str): A UUID representing the current owner of this partition.
+                - `last_modified_time` (float): The last time this ownership was claimed.
+                - `etag` (str): The Etag value for the last time this ownership was modified. Optional depending
+                  on storage implementation.
+        """
+        try:
+            partition_key = "{} {} {} Ownership".format(
+                fully_qualified_namespace, eventhub_name, consumer_group
+            )
+            partition_key_filter = "PartitionKey eq '{}'".format(partition_key)
+            entities = self._table_client.query_entities(partition_key_filter, **kwargs)
+            result = []
+            for entity in entities:
+                ownership = {
+                    "fully_qualified_namespace": fully_qualified_namespace,
+                    "eventhub_name": eventhub_name,
+                    "consumer_group": consumer_group,
+                    "partition_id": entity[u"RowKey"],
+                    "owner_id": entity[u"ownerid"],
+                    "last_modified_time": _to_timestamp(
+                        entity.metadata.get("timestamp")
+                    ),
+                    "etag": entity.metadata.get("etag"),
+                }
+                result.append(ownership)
+            return result
+        except Exception as error:
+            logger.warning(
+                "An exception occurred during list_ownership for "
+                "namespace %r eventhub %r consumer group %r. "
+                "Exception is %r",
+                fully_qualified_namespace,
+                eventhub_name,
+                consumer_group,
+                error,
+            )
+            raise
+
+    def list_checkpoints(
+        self, fully_qualified_namespace, eventhub_name, consumer_group, **kwargs
+    ):
```

Comment:
need a type hint here:
```suggestion
    ):
    # type: (str, str, str, Any) -> Iterable[Dict[str, Any]]
```